openmdao_book/features/core_features/working_with_components/continuous_variables.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>Missing or misspelled tags</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Added the correct tags.</p>